### DISCUSSION TITLE
How do you disable snapshots?

### DISCUSSION THREAD

**Discussion by roehnan** [dragonflydb/dragonfly#1691] [Q&A] [answered]
I've noticed that the v1.8.0 version behaves differently. We just had a Kubernetes pod crash with the following error:
`W20230810 08:52:11.857513    12 server_family.cc:629] Failed to perform snapshot Permission denied: Couldn't open file for writing (is direct I/O supported by the file system?)`

Our configuration doesn't specify any snapshot / save schedule.

```    Args:
      --alsologtostderr
      --cluster_mode=emulated
      --maxmemory=24gb
      --dbnum=1
      --cache_mode=true
      --version_check=false
```

**Comment by romange** [comment]
@roehnan this is not an error, this is a warning. I wonder what's the real reason for the crash. Can you paste the whole log?

**Comment by romange** [comment]
To your question, Dragonfly saves a snapshot on shutdown and the instance has been actively shutdown via a command or the process signal.

**Reply by romange** [reply, marked as answer]
Finally, you can force Dragonfly to omit snapshotting altogether by passing `--dbfilename=` in the flag file

**Comment by roehnan** [comment]
Thank you for the clarification. I think there was an error in our environment that caused the pod to shutdown and subsequently it was unable to write the snapshot. 

I don't think I can get at the logs now (since the pod was removed), but I will keep an eye out. Please close the issue and I'll reach out in Discord if I have any other information. Thanks!
